execution/binary/scalar.go

```diff
@@ -63,6 +64,7 @@ func NewScalar(
 		scalar:        scalar,
 		operation:     binaryOperation,
 		opName:        parser.ItemTypeStr[op],
+		isComparison:  op.IsComparisonOperator(),
```

Comment:
Maybe we can directly pass `op` in struct and use that instead `Explain` and `loadSeries`? 
```suggestion
		op:        op,
```